packages/manager/apps/dedicated/client/app/dedicated/server/details/server.service.js

```diff
@@ -2213,4 +2213,17 @@ export default class ServerF {
       .get(`/dedicated/server/${serviceName}/bringYourOwnImage`)
       .then(({ data }) => data);
   }
+
+  getServices(modifiedSince) {
+    return this.OvhHttp.get('/service', {
```

Comment:
If duable prefer to use `$http`.

Reply:
Its not, I tried.

Reply:
This code snippet isn't working?

```js
return this.$http.get('URL', {
  params: {
    external: false,
    type: '/dedicated/server',
   },
   serviceType: 'aapi',
   headers: {
     pragma: 'no-cache',
   },
  });
```

Reply:
@antleblanc yeah, i tried the same it didn't work. But let me give it one more shot.

Reply:
Taken care.